tests/parser/syntax/test_byte_string.py

```diff
@@ -1,26 +0,0 @@
-import pytest
-
-from vyper import compiler
-
-valid_list = [
-    """
-@external
-def foo() -> String[10]:
-    return "badminton"
-    """,
-    """
-@external
-def foo():
-    x: String[11] = "¡très bien!"
-    """,
-    """
-@external
-def test() -> String[100]:
-    return "hello world!"
-    """,
-]
```

Comment:
Duplicated tests: https://github.com/vyperlang/vyper/blob/a5995a91b769544b43819ef68ab6be5b6c6e1274/tests/parser/syntax/test_string.py#L5-L15

I moved the last example to this file.